socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
-            )
+            self.logger.debug('writing to %s' % (
+                hasattr(a_store, 'wrapped_object') and
+                a_store.wrapped_object or a_store.__class__
+            ))
+
+            self.quit_check()
+            try:
+                a_store.save_raw_and_processed(
+                    raw_crash,
+                    dump,
+                    processed_crash,
+                    crash_id
+                )
+            except Exception, x:
```

Comment:
The modern notation is `except Exception as x:` and I'd rather see `exception` or `exp` instead of just `x` as a variable name (...when it's not just inside a list comprehension loop).